gm4_liquid_minecarts/data/liquid_minecarts/functions/init_liquid.mcfunction

```diff
@@ -0,0 +1,19 @@
+#@s = empty liquid minecart next to a liquid tank pointed at it containing liquid
+#run from liquid_minecarts:try_to_unload
+
+data modify entity @s CustomName set from block ~ ~ ~ CustomName
+data modify entity @e[type=armor_stand,tag=gm4_liquid_minecart_display,limit=1,sort=nearest] HandItems[0] set from entity @e[type=armor_stand,tag=gm4_liquid_tank_display,limit=1,sort=nearest] ArmorItems[3]
+data modify entity @s Tags set from entity @e[type=armor_stand,tag=gm4_liquid_tank,limit=1,sort=nearest] Tags
+tag @s remove gm4_lt_idle

```

Comment:
```suggestion
```
This tag no longer exists in the Liquid Tanks base. Since functions run per tank, tanks that would be "idle" due to no items inside them simply don't run the processing functions

Reply:
When you say "no longer", is that a change you made since I created this PR? Cos tanks definitely had this tag while I was testing.

Reply:
I just looked into it. Turns out tanks were given this tag when created, but the tag was never used anywhere in the module. I've removed that tag from the 1.14 module now.